builder/default.conf

```diff
@@ -0,0 +1,23 @@
+server {
+  root /var/www;
+  index index.html;
+
+  location ~ ^/(?!static/)(?<type>(.*))$ {
+    add_header Vary "Accept";
+    try_files $prefix/$type$suffix $uri /index.html =404;
+  }
```

Comment:
This stanza is very important to making the registry work, so it would be nice if we didn't have to overwrite it as part of the internal server.

Reply:
The internal one will look exactly like this but with SSL enabled and a server name set so it gets matched before the default one.

Reply:
Could we put the include in this block instead so we can keep the config in the open and only inject specifically SSL config?

Reply:
I think it would be clearer with the current approach instead of having partial includes but I don't have a strong opinion on that.